lab/lab1/part3-data-transformation.html

```diff
@@ -55,9 +55,9 @@
     Often, our data will contain inconsistencies that can cause problems in our
     code. In the code below, switch out "bikeArrayClean" with "bikeArrayDirty".
     If you reload your map now, the markers should be gone. You can review the
-    dirty data in part4-data-dirty.js.
+    dirty data in part3-data-dirty.js.
 
-    Without modifying part4-data-dirty.js, change your code to account for the
+    Without modifying part3-data-dirty.js, change your code to account for the
```

Comment:
This was definitely an unfortunate oversight on Nathan and my part to not update these lab numbers.

But. This indeed is the challenging part of this lab: to "clean" the dirty data. You did not do this. This is the reason for my change requested flag.